kura/distrib/src/main/resources/Win32-nn/start_kura_debug.bat

```diff
@@ -0,0 +1,32 @@
+@echo off
+
+set KURA_ROOT=c:\opt\eclipse
+set KURA_HOME=%KURA_ROOT%\kura\kura
+set KURA_HOME_URI=/%KURA_HOME:\=/%
+set KURA_DATA=%KURA_ROOT%\kura\data
+set KURA_TEMP=\tmp\.kura
```

Comment:
But wouldn't it be much easier for a user to rely on OS specific behavior, like %TEMP% instead of requiring him to create a "/tmp" directory first. Which might not even be possible on some machines due to user access restrictions?